tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -1686,22 +1683,19 @@ static final boolean userCanDeleteUser(User managerUser, int targetId, int targe
     }
 
     static final boolean userCanDoList(User managerUser) {
-        return (userIsVetter(managerUser));
+        return managerUser.getLevel().canDoList();
     }
 
     public static final boolean userCanCreateUsers(User u) {
-        return (userIsTC(u) || userIsExactlyManager(u));
+        return u.getLevel().canCreateUsers();
     }
 
     static final boolean userCanSubmit(User u) {
         return userCanSubmit(u, SurveyMain.phase());
     }
 
     static final boolean userCanSubmit(User u, SurveyMain.Phase phase) {
-        if (phase == SurveyMain.Phase.READONLY) {
-            return false;
-        }
-        return ((u != null) && userIsStreet(u));
+        return u.getLevel().canSubmit(phase.getCPhase());
     }
```

Comment:
This method -- with two args -- appears to be unused except by TestCompatUserLevelDataDriven

so this is all much ado about not very much? if so, we could simply remove this method as well, and remove the call to it in the test